_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+  - In some special cases, the assigned reviewer may desire some level of input, advice, or guidance from a specific expert to facilitate review of the manuscript. In such instances, the reviewer must seek approval from the editor prior to sharing the contents of the manuscript with the expert. Following editorial approval, the assigned reviewer must clearly communicate the need for confidentiality to the consulted expert. The identity of the expert must also be disclosed to the editor at that time.  
+  - In some special cases, the reviewer may wish to disclose his/her identity to the manuscript authors, the editor should be consulted. There may be instances where the editor requests that the reviewer retains anonymity. The decision to disclose such information to authors is the sole discretion of the editor. However, such decisions should be accompanied by appropriate rationale from the editor. 
+  - Reviewers may disclose publicly that they have served as a reviewer for LiveCoMS. However, under no circumstances should the reviewer identify himself/herself as the reviewer of a specific manuscript. 
+- Reviewers should explain and support their judgments adequately so that editors and authors may understand the basis of their comments. Any statement that an observation, derivation, or argument had been previously reported should be accompanied by the relevant citation. Unsupported assertions by reviewers (or by authors in rebuttal) are of little value and should be avoided.
+- A reviewer should be alert to failure of authors to cite relevant work by other scientists, bearing in mind that complaints that the reviewer’s own research was insufficiently cited may seem self-serving. A reviewer should call to the editor’s attention any substantial similarity between the manuscript under consideration and any published paper or any manuscript submitted concurrently to another journal.
+- A reviewer should act promptly, submitting a report in a timely manner. Should a reviewer receive a manuscript at a time when circumstances preclude prompt attention to it, the unreviewed manuscript should be returned immediately to the editor. Alternatively, the reviewer might notify the editor of probable delays and propose a revised review date.
+- Reviewers should not use or disclose unpublished information, arguments, or interpretations contained in a manuscript under consideration, except with the consent of the author. If this information indicates that some of the reviewer’s work is unlikely to be profitable, the reviewer, however, could ethically discontinue the work. In some cases, it may be appropriate for the reviewer to write the author, with copy to the editor, about the reviewer’s research and plans in that area.
```

Comment:
This point doesn't really apply since everything should be publicly up on GitHub before it's being reviewed. So the reviewer should be able to talk about it publicly with anyone, I think.

Reply:
good point, eliminated.